operator/src/main/java/oracle/kubernetes/operator/steps/ReadHealthStep.java

```diff
@@ -108,6 +106,7 @@ public NextAction apply(Packet packet) {
     }
 
     private HttpRequest createRequest() {
+      LOGGER.fine("Create REST request to service URL: " + getRequestUrl());
```

Comment:
Not sure if this is going to be logged for every request which is executed every 3 seconds? This causes log files to be verbose, can it be FINER instead of FINE?